tests/test_ansible.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import unittest
+import shlex
+from helper import shell_process
+import re
+
+"""Moneo ansible single node test"""
+
+
+class AnsibleTestCase(unittest.TestCase):
+    """Ansible unit test class"""
+    
+    def check_docker_status(self):
+        """Helper to check docker container status"""
+        cmd='sudo docker container ls'
+        args = shlex.split(cmd)
+        result = shell_process.shell_cmd(args)
+        if ('grafana' in result) or ( 'prometheus' in result ):
+            return True #deployed
+        else:
+            return False #not deployed
```

Comment:
This is more of a stylistic choice which does not impact the functionality of the code, but you could rewrite lines 20-23 as: `return ('grafana' in result) or ( 'prometheus' in result )`